.travis.yml

```diff
@@ -123,12 +123,13 @@ deploy:
     local_dir: docs/
     on:
       branch: master
+
   - provider: script
     skip_cleanup: true
-    script: scripts/travis/deploy-codepush.sh
-    on:
-      branch: master
-      condition: $JS && $run_deploy == 1
+    script: rvm 2.3 do bundle exec fastlane codepush
+    # on:
+      # branch: master
+      # condition: $JS && $run_deploy == 1
```

Comment:
I need this to run on `master` once, then I'll disable it again once codepush is happy.